source/main/modules/window-manager/index.ts

```diff
@@ -324,7 +397,7 @@ export default class WindowManager {
    *
    * @return  {WindowPosition}                               A sanitised WindowPosition
    */
-  private _retrieveWindowPosition (type: string, defaultSize: Rect|null, predicate?: Record<string, any>): WindowPosition {
+  private _retrieveWindowPosition (type: string, defaultSize: Rect | null, predicate?: Record<string, any>): WindowPosition {
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.